Dockerfile.django

```diff
@@ -88,14 +89,21 @@ RUN \
   true
 USER root
 RUN \
-    adduser --system --no-create-home --disabled-password --gecos '' \
-        --uid ${uid} ${appuser} && \
+    # Set a known user and group with a higher number uid/git
+    addgroup --gid ${gid} ${appuser} && \
+    adduser --no-create-home --disabled-password \
+            --gecos 'DefectDojo' --uid ${uid} --gid ${gid} ${appuser} && \
     chown -R root:root /app && \
     chmod -R u+rwX,go+rX,go-w /app && \
+    # Allow for bind mounting local_settings.py and other setting overrides
+    chown root:${appuser} /app/dojo/settings && \
+    chmod 775 /app/dojo/settings && \
+    # Allow ${appuser} to run entrypoint scripts
+    chown root:defectdojo /entrypoint-* && \
```

Comment:
Why here not root:root and to go with 755?

Reply:
Since we have a defectdojo group (technically ${appuser} group), I thought it was more explicit to list the group as defectdojo with write permissions.  An `ls -lah /app/dojo/settings` will let you know that defectdojo is expected to write in this directory.  It's also the only non-root group owned directory so it's very explicit.